sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -87,6 +90,15 @@ void uhcp_handle_prefix(uint8_t *prefix, uint8_t prefix_len, uint16_t lifetime,
 
     gnrc_netapi_set(gnrc_wireless_interface, NETOPT_IPV6_ADDR, (64 << 8),
                     prefix, sizeof(ipv6_addr_t));
+#ifdef MODULE_GNRC_SIXLOWPAN_CTX
+    uint8_t cid = 0;
+
+    /* find first free context ID */
+    while (gnrc_sixlowpan_ctx_lookup_id(cid) != NULL) {
+        cid++;
```

Comment:
Whaht happens if the same prefix is being renewed a couple of times? would this exhaust the available ctx slots?

Reply:
You are right, will fix!

Reply:
Done